quickstarts/hello-rules/hellorules/pom.xml

```diff
@@ -24,7 +24,7 @@
   <properties>
     <jaxb-api.version>2.3.0.redhat-1</jaxb-api.version>
     <maven.compiler.version>3.8.0</maven.compiler.version>
-    <ba-platform-bom.version>7.13.4.redhat-00002</ba-platform-bom.version>
```

Comment:
Just a note that version 7.13.5.redhat-00001 is somehow speculative, as the BA BOM could have different prod version as we may have to rebuild it more than one time. Said that, maybe it is better to keep the older version for now (7.13.4.redhat-00002) so in the future we remember to properly update it?

Reply:
@spolti wdyt? I guess we should keep the old versions for now otherwise I'm afraid we will forget to update this in the future in case the bom version would be different from `7.13.5.redhat-00001`

Reply:
That's a good question, at least if we keep the previous version, it will still work in case of miss the next version update.
let me create a jira for this, and, later, keep the 7.13.4.

Reply:
@spolti I agree, please create a Jira so we remember to update it for 7.13.5 and for now let's keep the 7.13.4.